bridge/github/config.go

```diff
@@ -205,13 +205,42 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
 	for {
-		fmt.Print("Github project URL: ")
 
-		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println("The remotes are:")
+
+		repos, nil := repo.ListRemotes()
+
+		i := 0
+		for key, value := range repos {
```

Comment:
you need to filter the remotes and suggest only the one that can work here, that is, the remotes from github